MastodonSDK/Sources/MastodonSDK/Entity/Mastodon+Entity+Account.swift

```diff
@@ -84,6 +84,64 @@ extension Mastodon.Entity {
     }
 }
 
+extension Mastodon.Entity.Account: Hashable {
+    public func hash(into hasher: inout Hasher) {
```

Comment:
I'm wondering if taking properties such as `followersCount` et. al. might lead to differences on the same account because of stale local data. Maybe `id`, `username`, `acct`, `url` are enough?

Reply:
This is a good, valid point. What makes things even worse is that depending on the server you ask, the same user has a different ID. Take [me on mastodon.social](https://mastodon.social/api/v1/accounts/lookup?acct=zeitschlag@chaos.social) and [me on chaos.social](https://chaos.social/api/v1/accounts/lookup?acct=zeitschlag@chaos.social):

<img width="762" alt="Screenshot 2023-11-14 at 12 06 05" src="https://github.com/mastodon/mastodon-ios/assets/2580019/11fcc544-3f44-4a59-a48f-71530e947e5b">

Reply:
Yes, I think we had a method which always gave us the full account handle (incl. server) I think we should compare those and I wouldn't take the other attributes into consideration.

Reply:
You mean `acctWithDomainIfMissing(_:)`, right? That still uses the local domain (and you'd need to bring the local domain in `==` or `hash_into`. Luckily, there's a computed property `domain` which uses `url`, so I decided to compare `url` directly in both `Hashable` and `Equatable`.